ds3-sdk/src/main/java/com/spectralogic/ds3client/utils/ResponseUtils.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2015 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3client.utils;
+
+import com.google.common.collect.ImmutableList;
+
+public class ResponseUtils {
+    public static ImmutableList<Integer> toImmutableIntList(final int[] expectedStatuses) {
+        final ImmutableList.Builder<Integer> integerBuilder = ImmutableList.builder();
+
+        for (final int status : expectedStatuses) {
```

Comment:
since this is a util function I can assume that the user is using it right, and if not this is on him...
am I right?